### PR TITLE
Update apache drill to 1.0.0 release

### PR DESCRIPTION
**Pull request by LarsFronius**

### CODE REVIEW

File: Library/Formula/apache-drill.rb [hunk 1]
```diff
@@ -1,16 +1,14 @@
 class ApacheDrill < Formula
   homepage "https://drill.apache.org/download/"
-  url "https://www.apache.org/dyn/closer.cgi?path=drill/drill-0.9.0/apache-drill-0.9.0.tar.gz"
-  sha256 "1680b4b937d26623f0f01d7719b0809decf3341f445f5caea3feed1ecbc16c4d"
+  url "https://www.apache.org/dyn/closer.cgi?path=drill/drill-1.0.0/apache-drill-1.0.0.tar.gz"
+  sha1 "0b43d52f6754bad7c7bb3c36e0dfc4c9de6d71e8"
```
Comment (bfontaine) [inline Library/Formula/apache-drill.rb]: Please use a `sha256` instead of a `sha1`.

File: Library/Formula/apache-drill.rb [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
     libexec.install Dir["*"]
-    ["drill_dumpcat", "runbit", "sqlline"].each do |bin_file|
-      bin.write_exec_script Dir["#{libexec}/bin/#{bin_file}"]
-    end
+    bin.write_exec_script Dir["#{libexec}/bin/*"]
   end
 
   test do
-    system "#{bin}/sqlline <<< 'SELECT * FROM cp.`employee.json`;'"
+    system "#{bin}/sqlline -u jdbc:drill:zk=local <<< '!tables'"
```
Comment (bfontaine) [inline Library/Formula/apache-drill.rb]: Use `pipe_output("#{bin}/sqlline -u jdbc:drill:zk=local", "!tables", 0)`.

**Comment by LarsFronius** [comment]
Hey @bfontaine - incorporated your feedback now. Good to go?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `apache-drill 1.0.0`.

**Pull request closed without merging**